art/project.py

```diff
@@ -103,7 +168,6 @@ def check_if_must_be_run(self, step: "Step", checks: List[Check]) -> bool:
             try:
                 self.check_checks(step, checks)
             except Exception as e:
-                art_logger.exception(e)
```

Comment:
Why don't raise an exception?

Reply:
Exception was not raised since long ago. Here I just removed logging it as it was causing a very big print statement every time Experiment was run.

Reply:
okay, looks good